spec/features/user_registration_spec.rb

```diff
@@ -23,7 +23,7 @@
       fill_in "Password", with: @user_info[:password]
       fill_in "Confirm Password", with: @user_info[:password]
 
-      click_button "Register New User"
+      click_button "Create User"
```

Comment:
Not super fond of the knock-on effect of using a partial here. We should look into rendering the submit button within the partial using different text depending on the context. "Create User" and "Update User" are just weird phrases to present _to_ the user. This shouldn't block a merge, but I'd like to vote in favor of adding it as an issue for later refactoring.